10-public-offer.md

```diff
@@ -0,0 +1,141 @@
+
+ * Magic bytes is an array of bytes: `[0x46, 0x43, 0x53, 0x57, 0x41, 0x50]`, corresponding to `FCSWAP` in ASCII
+ * The version and list of activated features as a two bytes unsigned little endian integer, currently set as 1, i.e. `[0x01, 0x00]`
+ * The network as one byte: `0x01` for mainnet, `0x02` for testnet, and `0x03` for local
+ * The arbitrating asset identifier followed by the accordant identifier, two four bytes unsigned integer serialized in little endian
+ * The arbitrating asset amount followed by the accordant amount
+    * one unsigned integer byte defining the number of following bytes to parse
+    * an array of bytes `[bytes]` representing the amount in the smallest granular native unit for its respective blockchain; bytes are serialized with the native blockchain consensus rules.
+ * The cancel timeout followed by the punish timeout
+    * one unsigned integer byte defining the number of following bytes to parse
+    * an array of bytes `[bytes]` representing the timelock value for its respective blockchain; bytes are serialized with the native blockchain consensus rules.
+ * The fee strategy as one byte:
+    * `0x01` for the fixed fee, followed by
+        * one unsigned integer byte defining the number of bytes to parse per value
+        * an array of bytes `[bytes]` representing the value of the fixed fee interpreted for its respective blockchain; bytes are serialized with the native blockchain consensus rules.
+    * `0x02` for the range fee, followed by
+        * one unsigned integer byte defining the number of bytes to parse per value
+        * an array of bytes `[bytes]` representing the value of minimum fee interpreted for its respective blockchain; bytes are serialized with the native blockchain consensus rules.
+        * one unsigned integer byte defining the number of bytes to parse per value
+        * an array of bytes `[bytes]` representing the value of maximum fee interpreted for its respective blockchain; bytes are serialized with the native blockchain consensus rules.
+ * The future maker swap role as one byte: `0x01` for Alice and `0x02` for Bob
+
+```
+< [0x46, 0x43, 0x53, 0x57, 0x41, 0x50] MAGIC BYTES > < [u16] version > < [u8] network >
+< [u8; 4] arbitrating identifier > < [u8; 4] accordant identifier >
+< [u8] len > < [u8; len] arbitrating amount value >
+< [u8] len > < [u8; len] accordant amount value >
+< [u8] len > < [u8; len] cancel timeout value >
+< [u8] len > < [u8; len] punish timeout value >
+< [u8] fee strategy > < [u8] len > < [u8; len] fixed or minimum value > (< [u8] len > < [u8; len] maximum value >)
+< [u8] future maker role >
+```
+
+### Amounts
+
+For Bitcoin, the amounts must be serialized as a eight bytes unsigned little endian integer representing the number of satoshi.
+
+For Monero, the amounts must be serialized as a eight bytes unsigned little endian integer representing the number of monerujo.
```

Comment:
not atomic units? 1e-12 XMR

Reply:
You are right, 1e-12 is the atomic unit and is called piconero and not monerujo, will fix that

Reply:
Fixed in 2c1e4be82f010fc66a25e328e5f52a717cb26784